core/src/main/scala/akka/persistence/r2dbc/internal/BySliceQuery.scala

```diff
@@ -71,6 +152,13 @@ import org.slf4j.Logger
         toTimestamp: Option[Instant],
         behindCurrentTime: FiniteDuration,
         backtracking: Boolean): Source[SerializedRow, NotUsed]
+
+    def eventCountBuckets(
+        entityType: String,
+        minSlice: Int,
+        maxSlice: Int,
+        fromTimestamp: Instant,
+        limit: Int): Future[Seq[(Long, Long)]]
```

Comment:
Maybe do a case class for this vs tuple long long for clarity?

Reply:
done